pkg/types/openstack/machinepool.go

```diff
@@ -1,5 +1,15 @@
 package openstack
 
+// AllowedServerGroupPolicies is the set of allowed values for the
+// "ServerGroupPolicy" property of the MachinePool. The value will be set on
+// the Nova server group that will contain the machines.
+var AllowedServerGroupPolicies = map[ServerGroupPolicy]struct{}{
```

Comment:
Move this to a private variable in the ` pkg/types/openstack/validation` package.